src/js/components/landing/LandingContent.jsx

```diff
@@ -26,24 +26,30 @@ export default class LandingContent extends React.Component {
     constructor(props) {
         super(props);
 
+        this.didUnmount = false;
+
         this.state = {
             expanded: false,
+            type: this.props.type,
```

Comment:
the reason is because when you go from `landing` to `detachedLanding` the content wont change since the component isn't being re-rendered.